Part_2/UIandFriends/Ticket.java

```diff
@@ -0,0 +1,28 @@
+package Final_Project;
+
+public class Ticket extends Flight {
+	private String ticketId;
+	private String firstName;
+	private String lastName;
+	
+	public Ticket (String fId, String dest, String src, String dateOF, String timeOF, String durOF, String tS, String tA, String prc, String tId, String fN, String lN){
+		super(fId, dest, src, dateOF, timeOF, durOF, tS, tA, prc);
+		ticketId = tId;
+		firstName = fN;
+		lastName = lN;
+	}
+	

```

Comment:
Can we have a constructor in the form `public Ticket(Flight flight, String ticketID, String firstName, String lastName)` pls